test/functional/page_objects/common_page.ts

```diff
@@ -371,6 +371,12 @@ export function CommonPageProvider({ getService, getPageObjects }: FtrProviderCo
       await browser.pressKeys(browser.keys.ENTER);
     }
 
```

Comment:
This is nice little helper for debugging tests when needed, left for future users